.vscode/tasks.json

```diff
@@ -2,6 +2,16 @@
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
   "version": "2.0.0",
+  "windows": {
```

Comment:
What does this actually control? Does it control what terminal is used when I hit `` ctrl+` `` on Windows? Or just what shell these tasks run in?

Reply:
Just what shell is used to run tasks in.